optimize/util/optimize-commons/src/main/java/io/camunda/optimize/dto/optimize/SimpleDefinitionDto.java

```diff
@@ -26,19 +25,16 @@ public class SimpleDefinitionDto {
   @EqualsAndHashCode.Include @NonNull private String key;
   private String name;
   @EqualsAndHashCode.Include @NonNull private DefinitionType type;
-  @JsonIgnore private Boolean isEventProcess = false;
```

Comment:
I think I already commented in a subclass before, but this is a breaking API change that needs handling accordingly

Reply:
This is also included in the `Required FE changes not yet added to this branch:`  section